### PR TITLE
renamed Remove & altered popover description

### PR DESCRIPTION
**Pull request by eugen-soliar**
# Description

- Renamed 'Remove' button in dropdown menu.
- Altered description in popover dialog accordingly.
Fixes [1009](https://github.com/usebruno/bruno/issues/1009)

CloseCollection
CloseCollectionPopup

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

File: packages/bruno-app/src/components/Sidebar/Collections/Collection/RemoveCollection/index.js [hunk 1]
```diff
@@ -10,15 +10,16 @@ const RemoveCollection = ({ onClose, collection }) => {
...
-        toast.success('Collection removed');
+        toast.success('Collection disconnected');
         onClose();
       })
-      .catch(() => toast.error('An error occurred while removing the collection'));
+      .catch(() => toast.error('An error occurred while disconnecting the collection'));
   };
 
   return (
     <Modal size="sm" title="Remove Collection" confirmText="Remove" handleConfirm={onConfirm} handleCancel={onClose}>
```
Comment (Its-treason) [inline packages/bruno-app/src/components/Sidebar/Collections/Collection/RemoveCollection/index.js]: The title and confirmText should also be updated

File: packages/bruno-app/src/components/Sidebar/Collections/Collection/RemoveCollection/index.js [hunk 2]
```diff
@@ -10,15 +10,16 @@ const RemoveCollection = ({ onClose, collection }) => {
...
-      Are you sure you want to remove collection <span className="font-semibold">{collection.name}</span> ?
+      Are you sure you want to disconnect collection <span className="font-semibold">{collection.name}</span> from
+      Bruno? It will remain in your file system anyways.
```
Comment (Its-treason) [inline packages/bruno-app/src/components/Sidebar/Collections/Collection/RemoveCollection/index.js]: I think the last sentence sounds weird with the "anyways". How about: "The collection files will remain in your file system in [Collection Path here]"

**Review by Its-treason — CHANGES_REQUESTED** [review]
I think we should change the naming here, but I'm not sure if "disconnect" is the best wording, how about "close"? I'm not entirely sure about this, but I never saw something like this named "disconnect".

**Comment by helloanoop** [comment]
Merged! Thank you @eugen-soliar !
Thanks for the review @Its-treason !

I also made some small UX updates
image

**Pull request merged**